### PR TITLE
allows you to wirecutters the manifests off crates without opening them

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

title

## Why It's Good For The Game

I was asked to make this PR, I guess it's a QoL?

## Changelog
:cl:
add: crowbarring manifests off crates
/:cl:

### CODE REVIEW

**Comment by Owai-Seek** [comment]
Just adding to this since Kev wants to know why it would be good. This would (kinda?) be a port from TG as a QoL, I saw it while browsing through their PR's and thought it would be nice, especially for all the crates that have an enormous amount of items in them where the paper gets lost. Makes gathering papers to ship back easier.

https://github.com/tgstation/tgstation/pull/19164

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 1]
```diff
@@ -45,6 +45,13 @@
 	if(manifest)
 		tear_manifest(user)
 
+/obj/structure/closet/crate/tool_interact(obj/item/W, mob/user)
+	if(istype(W, /obj/item/wirecutters))
+		if (manifest)
+			tear_manifest(user)
+			return TRUE
+	return ..()
+
```
Comment (Ghommie) [inline code/game/objects/structures/crates_lockers/crates.dm]: Any reason not to use `wirecutter_act` and skipping the typecheck?

Comment (Seris02) [reply code/game/objects/structures/crates_lockers/crates.dm]: I tried it and it didn't work, so yes

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 2]
```diff
@@ -45,6 +45,13 @@
 	if(manifest)
 		tear_manifest(user)
 
+/obj/structure/closet/crate/tool_interact(obj/item/W, mob/user)
+	if(istype(W, /obj/item/wirecutters))
```
Comment (Ghommie) [inline code/game/objects/structures/crates_lockers/crates.dm]:
```suggestion
	if(W.tool_behaviour == TOOL_WIRECUTTER && manifest)
```
Also remove the line below.

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Review by Ghommie — APPROVED** [review]

**Pull request merged**